fe/src/main/java/org/apache/doris/planner/SchemaScanNode.java

```diff
@@ -98,16 +99,17 @@ protected void toThrift(TPlanNode msg) {
         if (schemaWild != null) {
             msg.schema_scan_node.setWild(schemaWild);
         }
-        if (user != null) {
-            msg.schema_scan_node.setUser(user);
-        }
+
         ConnectContext ctx = ConnectContext.get();
         if (ctx != null) {
             msg.schema_scan_node.setThread_id(ConnectContext.get().getConnectionId());
         }
         msg.schema_scan_node.setIp(frontendIP);
         msg.schema_scan_node.setPort(frontendPort);
-        msg.schema_scan_node.setUser_ip(userIp);
+
+        UserIdentity currentUser = ConnectContext.get().getCurrentUserIdentity();
+        msg.schema_scan_node.setUser(currentUser.getQualifiedUser());
+        msg.schema_scan_node.setIp(currentUser.getHost());
```

Comment:
I will refactor this section. After that, the `user` and `user_ip` are deprecated, and be replaced by a new member `current_user_ident`, which is type of `TUserIdentity`.
Add a new type `TUserIdentity` because a user identity is not only composed by `user` and
`ip`, but also has a property `is_domain`, to support BNS of DNS.